hooks/declarative-subsequent-scans/hook.ts

```diff
@@ -127,3 +129,16 @@ function generateCascadingScanName(
   }
   return `${namePrefix}-${cascadingRule.metadata.name}`;
 }
+
+function wildcardMatcher(
+  findingValue: any,
+  matchesRuleValue: any
+) : boolean {
+  if(isString(matchesRuleValue)){
+    try{
+      return new RegExp('^' + new String(matchesRuleValue).replace(/\*/g, '.*') + '$').test(findingValue);
```

Comment:
I think it might be worth it to delegate the wildcard matching to a library, as this is quite hard to get completely right, for example the might be cases where you'd want to escape the `*` and have it match the character not as a wildcard.

I think a good lib for it would be the `isMatch` function from https://www.npmjs.com/package/matcher

Reply:
Thank you for suggesting that lib, this has been implemented in https://github.com/secureCodeBox/secureCodeBox/pull/374/commits/5884c89ede46c33716bec800f3482c3feefeefcb which is included in this PR.

I've noticed that there is a parameter to make the matching case sensitive. I've set it to true to match current application behavior, but I can see a case for having the CascadingRules be case-insensitive.

Reply:
Awesome thank you 👍